next/apps/Butler.py

```diff
@@ -134,6 +137,23 @@ def __init__(self, collection, uid_prefix, exp_uid, db, timing=True):
         self.timing = timing
         self.memory = Memory(collection, exp_uid)
 
+    def timed(op_type='set'):
```

Comment:
The attributes that `@timed` stores timing data to are `Collection`-specific, which is why I encapsulated it in `Collection`.